python/ray/train/tensorflow.py

```diff
@@ -79,3 +81,19 @@ def handle_failure(self, worker_group: WorkerGroup,
             worker_group.execute(shutdown_session)
         worker_group.add_workers(len(failed_worker_indexes))
         self.on_start(worker_group, backend_config)
+
+
+def prepare_dataset_shard(tf_dataset_shard: tf.data.Dataset):
```

Comment:
Could you add this to the [API reference](https://github.com/ray-project/ray/blob/master/doc/source/train/api.rst)?